multiqc/modules/custom_content/custom_content.py

```diff
@@ -202,10 +202,12 @@ def dict_constructor(loader, node):
                         hs[k] = v
                 gsheaders = hs
 
-            # Add namespace if not specified
-            for k in gsheaders:
-                if 'namespace' not in gsheaders[k]:
-                    gsheaders[k]['namespace'] = c_id
+            # Add namespace and description if not specified
+            for h in gsheaders:
+                if 'namespace' not in gsheaders[h]:
+                    gsheaders[h]['namespace'] = mod['config'].get('namespace', k)
```

Comment:
If you're going to switch `k` for `h`, make sure you do it everywhere 😉

Reply:
hehe.. I was expecting this 😅

Reply:
ah ha.. no, the reason why I switched for `h` is to be able to use `k` from the main `for` loop _scope_. Because earlier you tried to use `c_id`, I assume you wanted to use the "custom module" name rather than the "column name" _(which I too prefer)_. 

`c_id` would not work as intended as its `for` loop end before and it will just contain the module name from last iteration.

Reply:
ahhh, ok cool - good spot! 👍